webpack.config.js

```diff
@@ -138,27 +130,8 @@ module.exports = {
   },
 };
 
-/**
- * Utils
- */
-
-function targetBuild() {
-  // target BUILD parameter is case insensitive (default chrome)
-  const interpretTargetBuild = (requested = '') => {
-    return Object.keys(BUILD)
-      .find(build => build == requested.toUpperCase())
-      || BUILD.CHROME;
-  }
-
-  // grab target build parameter (passed as command arg)
-  return interpretTargetBuild(process.env.BUILD);
-}
-
-function manifestFiles() {
-  return [
-    // base manifest file
-    'manifest/base.manifest.json',
-    // each build can extend the base manifest with a file of this form
-    `manifest/${targetBuild().toLowerCase()}.manifest.json`,
-  ];
+function stringifyValues(obj) {
```

Comment:
You could perhaps move this to the `build.config.ts` file to keep the webpack file a little cleaner.